sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1288,3 +1288,13 @@ def analyzer(doc):
 
     with pytest.raises(Exception, match="testing"):
         Estimator(analyzer=analyzer, input='file').fit_transform([f])
+
+
+def test_CountVectorizer_fit_warning():
```

Comment:
function names need to be all lowercase

This could also be,
```
@pytest.mark.parametrize("Vectorizer", [CountVectorizer, TfidfVectorizer, HashingVectorizer])
def test_unused_parameters_warn(Vectorizer):
    vect = Vectorizer()
```

Reply:
changed the function name for now ...will parameterize it later tonight

Reply:
Made the function parameterized . Have only included [CountVectorizer] for now. Will add the others as I add warnings for them  .